mixer/pkg/cache/lruCache.go

```diff
@@ -230,8 +236,9 @@ func (c *lruCache) SetWithExpiration(key interface{}, value interface{}, expirat
 	ent.value = value
 	ent.expiration = exp
 
-	c.stats.Writes++
 	c.Unlock()
+
+	atomic.AddUint64(&c.stats.Writes, 1)
```

Comment:
There are other places in the code where we are doing `c.stats.Writes++`. We should change them too.

Reply:
I don't see other places in lru_caches where we update the Writes field.